xml/System.Diagnostics/Activity.xml

```diff
@@ -973,10 +974,14 @@ The root ID is a substring from the <xref:System.Diagnostics.Activity.Id> or the
         <Parameter Name="value" Type="System.String" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">The baggage key name</param>
+        <param name="value">The baggage value mapped to the input key</param>
+        <summary>Add or update the Activity baggage with the input key and value.
+            If the input value is <see langword="null" /> - if the collection has any baggage with the same key, then this baggage will get removed from the collection.
+                - otherwise, nothing will happen and the collection will not change.
+            If the input value is not <see langword="null" /> - if the collection has any baggage with the same key, then the value mapped to this key will get updated with the new input value.
+                - otherwise, the key and value will get added as a new baggage to the collection.</summary>
```

Comment:
would make sense moving this section to the remarks? I am ok either way. just checking.

Reply:
I'm fine as-is, but if anyone champions that it should be changed I am happy to do so.